backend/internal/controller/server/msg_handler.go

```diff
@@ -66,3 +67,32 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 func (s server) ResetScoreHandler(w http.ResponseWriter, r *http.Request) {
 	s.game.ResetScore()
 }
+
+// ManipulateScoreHandler is handler for manipulation of the score.
+// Incoming URL should be in the format: '/goal?action=[add/sub]&team=[1/2]'.
+// Team ID 1 stands for white and 2 for Blue.
+func (s server) ManipulateScoreHandler(w http.ResponseWriter, r *http.Request) {
+
+	team := r.URL.Query().Get("team")
+
+	teamID, err := strconv.Atoi(team)
+	if err != nil || !(teamID == 1 || teamID == 2) {
```

Comment:
I'd consider extracting `!(teamID == 1 || teamID == 2)` to a helper function, because it's a bit hard to read, something like `isValidTeam`, but this is good too